scripts/add_brownfield.py

```diff
@@ -292,6 +292,38 @@ def update_heat_pump_efficiency(n: pypsa.Network, n_p: pypsa.Network, year: int)
     )
 
 
+def update_dynamic_ptes_capacity(n, n_p, year):
```

Comment:
```suggestion
def update_dynamic_ptes_capacity(n: pypsa.Network, n_p: pypsa.Network, year: int) -> None:
```
We want more type hints for new code :)